content/docs/pipelines/installation/overview.md

```diff
@@ -0,0 +1,96 @@
++++
+title = "Installation Options for Kubeflow Pipelines"
+description = "Overview of the ways to deploy Kubeflow Pipelines"
+weight = 10
++++
+
+Kubeflow Pipelines offers a few installation options.
+This page describes the options and the features available
+with each option:
+
+* Kubeflow Pipelines as [part of a full Kubeflow deployment](#full-kubeflow).
+* A [standalone](#standalone) Kubeflow Pipelines deployment.
+* **Alpha**: Kubeflow Pipelines deployed [from the
+  Google Cloud Platform (GCP) Marketplace](#marketplace).
+
+<a id="full-kubeflow"></a>
+## Full Kubeflow deployment
+
+Use this option to deploy Kubeflow Pipelines to your local machine, on-premises, 
+or to a cloud, as part of a full Kubeflow installation.
+
+Installation guide
+: [Kubeflow installation guide](/docs/started/getting-started/)
+
+Interfaces
+:
+  * Kubeflow UI
+  * Kubeflow Pipelines UI within or outside the Kubeflow UI
+  * Kubeflow Pipelines SDK
+  * Kubeflow Pipelines API
+  * Other Kubeflow APIs
+
+Notes on specific features
+: After deployment, your Kubernetes cluster includes all the 
+  [Kubeflow components](/docs/components/).
+  For example, you can use the Jupyter notebook services 
+  [deployed with Kubeflow](/docs/notebooks/) to create one or more notebook 
+  servers in your Kubeflow cluster.
+
+<a id="standalone"></a>
+## Kubeflow Pipelines Standalone
+
+Use this option to deploy Kubeflow Pipelines to your local machine, on-premises,
+or to a cloud, without the other components of Kubeflow.
+
+Installation guide
+: [Kubeflow Pipelines Standalone deployment 
+  guide](/docs/pipelines/standalone-deployment-gcp/)
+
+Interfaces
+: 
+  * Kubeflow Pipelines UI
+  * Kubeflow Pipelines SDK
+  * Kubeflow Pipelines API
+
+
+Notes on specific features
+: After deployment, your Kubernetes cluster contains Kubeflow Pipelines only. 
+  It does not include the other Kubeflow components. 
+  For example, to use a Jupyter Notebook, you must use a separate notebook 
+  server.
```

Comment:
Suggestion: add ", for example a local notebook or a hosted notebook in a cloud service such as the Google AI Platform Notebook" https://console.cloud.google.com/ai-platform/notebooks

Reply:
Nice. Done.